includes/wf_crm_webform_base.inc

```diff
@@ -186,6 +190,36 @@ abstract class wf_crm_webform_base {
   }
 
   /**
+   * Reorder returned results according to settings chosen in wf_civicrm backend
+   *
+   * @param string $ent
+   * @param array $values
+   * @return array $reorderedArray
+   * @todo Handle user-select case
+   */
+  protected function reorderByLocationType($ent, $values = array()){
+    $reorderedArray = array();
+    // For every location field setting
+    foreach ($this->settings['data']['contact'] as $setting) {
+      // For every Entity
+      foreach($setting[$ent] as $e){
+        // For every result row
+        foreach($values as $key => $value){
```

Comment:
Hello @colemanw 
I am not sure what you mean by that as `$setting` contains the settings chosen by the user on webform_civicrm tab and `$values` contain the data fetched based on those settings.